api/app/db/crud.py

```diff
@@ -185,6 +243,17 @@ def get_prediction_model(session: Session, abbreviation: str, projection: str) -
         filter(PredictionModel.projection == projection).first()
 
 
+def get_prediction_model_run_timestamp_records(
+        session: Session, complete: bool = True, interpolated: bool = True):
+    """ Get prediction model run timestamps (filter on complete and interpolated if provided.) """
+    query = session.query(PredictionModelRunTimestamp)
+    if interpolated is not None:
+        query = query.filter(PredictionModelRunTimestamp.interpolated == interpolated)
+    if complete is not None:
```

Comment:
https://www.python.org/dev/peps/pep-0008/#programming-recommendations - using 'is not None' is recommended as per PEP8

Reply:
@Kyubinhan also - reason I wrote it this way, is you get 3 different kinds of behaviour. If I specific "interpolated = True" - I can filter on when interpolation is True, if I specify "interpolated = False" I can filter on when interpolation is False, if I specify "interpolated = None", then I don't filter on those at all, and get all records regardless.

Reply:
Well, the context is a bit different. Both `interpolated` and `complete` are always boolean. Actually, these if statements are meaningless since they always have default values right?

Reply:
It's not always boolean, I could call `get_prediction_model_run_timestamp_records(session, complete=None, interpolated=None)` to get all records.

Reply:
hmmm good question. Let's ask @Sybrand - he wrote this part

Reply:
Okay so then we should change the params so that it won't set these two values True when you pass None, right?

Reply:
@Kyubinhan I'm not sure I understand the question. The values aren't set to True when you pass None. They default to True if you pass nothing. If you pass None, they're None?

Reply:
Case is closed :)

Reply:
@Kyubinhan @Sybrand so we're happy with this as is?

Reply:
I believe so.